config/manager/manager.yaml

```diff
@@ -44,6 +44,8 @@ spec:
           # use this node to create storage during setup phase
           - name: RABBIT_NODE
             value: "0"
+          - name: NNF_EC_COMMAND_TIMEOUT
+            value: "90s"
```

Comment:
For consistency with the k8s yaml below (i.e. `initialDelaySeconds` and in other places in rabbit), it might be better to name this `NNF_EC_COMMAND_TIMEOUT_SECONDS` and drop the `s` from `90s`.

Reply:
That's what I had originally and I then I switched it to use time.ParseDuration() which can take different units ("ms", "s", "m", "h", etc.). If you think consistency with the other fields is more important I can switch it back. Seconds is the most realistic unit for this timeout.

Reply:
The duraation strings definitely handy, but if seconds makes the most sense here then I think consistency should win.